lease/lessor_test.go

```diff
@@ -47,16 +63,21 @@ func TestLessorGrant(t *testing.T) {
 // the DeleteableKV.
 // The revoked lease cannot be got from Lessor again.
 func TestLessorRevoke(t *testing.T) {
+	dir, be := NewTestBackend(t)
+	defer be.Close()
+	defer os.RemoveAll(dir)
```

Comment:
This shouldn't matter for testing, but would it be clearer to `Close` first and then `RemoveAll` like:

 ```
defer os.RemoveAll(dir)
defer be.Close()
```

?


Below, it removes first and close might fail silently?

 ```
defer be.Close()
defer os.RemoveAll(dir)
```